Fix API docs in Auth module (#22565)
ballerina-platform/ballerina-lang: stdlib/auth/src/main/ballerina/src/auth/Module.md

> This module provides default authentication provider configurations that can be extended to create new authentication providers.

Reviewer: ```suggestion
This module provides the default authentication provider configurations, which can be extended to create new authentication providers.
```

This module provides default authentication provider configurations that can be extended to create new authentication providers.

> ### Inbound Authentication Provider

Reviewer 2: @Bhashinee shall we keep these technical details. I think it is worth to have them in API Docs

    password: "123"
});
```

For information on the operations, which you can perform with this module, see the below **Functions**. For examples on the usage of the operations, see the [

> Secured Service with Basic Auth Example](https://ballerina.io/v1-2/learn/by-example/secured-service-with-basic-auth.html), and [

Reviewer: ```suggestion
Secured Service with Basic Auth Example](https://ballerina.io/v1-2/learn/by-example/secured-service-with-basic-auth.html) and [
```

> For information on the operations, which you can perform with this module, see the below **Functions**. For examples on the usage of the operations, see the [
> Secured Service with Basic Auth Example](https://ballerina.io/v1-2/learn/by-example/secured-service-with-basic-auth.html), and [
> Secured Client with Basic Auth Example](https://ballerina.io/v1-2/learn/by-example/secured-client-with-basic-auth.html).

Reviewer 3: ```suggestion
For information on the operations, which you can perform with this module, see the below **Functions**. For examples on the usage of the operations, see the [Secured Service with Basic Auth Example](https://ballerina.io/v1-2/learn/by-example/secured-service-with-basic-auth.html), and [Secured Client with Basic Auth Example](https://ballerina.io/v1-2/learn/by-example/secured-client-with-basic-auth.html).
```
